src/code/fv-components/stable-counter.js

```diff
@@ -0,0 +1,17 @@
+import React, {PropTypes} from 'react';
+
+const FVStableCounter = ({count, maxCount=5}) => {
```

Comment:
This component is close to being just a wrapper...but I expect it can be reused in stables for other challenges as well